tiledb/sm/c_api/tiledb.h

```diff
@@ -1035,6 +1035,14 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config);
  *    vfs.s3.max_parallel_ops` bytes will be buffered before issuing multipart
  *    uploads in parallel. <br>
  *    **Default**: 5MB
+ * - `vfs.s3.ca_file` <br>
+ *    Path to SSL/TLS certificate file to be used by cURL for for S3 HTTPS
+ * encryption. Follows cURL conventions: https://curl.haxx.se/docs/manpage.html
```

Comment:
Annoying, but would you mind fixing the indentation on these comment lines (and the other places where the config param is documented)?